virtual-server/examples/kubectl/README.md

```diff
@@ -4,6 +4,10 @@ To run any of the examples issue: `kubectl apply -f <manifest_file_name.yaml>`
 
 - [virtual-server-direct-attach-lb.yaml](virtual-server-direct-attach-lb.yaml) shows how to directly attach the Load Balancer IP to a Virtual Server. This will give 
 
+- [virtual-server-windows-internal-ip-only.yaml](virtual-server-windows-internal-ip-only.yaml) creates a Windows Virtual Server with no public IP - useful for servers that will only be accessed in your namespace.
```

Comment:
Clarify that this gets an internal STATIC ip (due to direct attach) and is useful for stuff like AD controllers.

Reply:
Good call - updated in 7163997